cmake/Platform/Linux/Configurations_linux.cmake

```diff
@@ -20,6 +20,33 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
             -msse4.1
     )
     ly_set(CMAKE_CXX_EXTENSIONS OFF)
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+
+    include(cmake/Platform/Common/GNU/Configurations_gnu.cmake)
```

Comment:
GNU isn't actually a compiler. It is actually the name of a [collection of tools](https://en.wikipedia.org/wiki/GNU) like LLVM is the the name of a [collection of tools](https://en.wikipedia.org/wiki/LLVM) as well.

I think CMake shouldn't have the Compiler Id be called GCC from the git-go, but I guess they went with GNU because there were so many variants of GCC at the time they added support